lib/libgssapi/gss_add_oid_set_member.3

```diff
@@ -107,12 +107,12 @@ others, and derivative works that comment on or otherwise explain it
 or assist in its implementation may be prepared, copied, published
 and distributed, in whole or in part, without restriction of any
 kind, provided that the above copyright notice and this paragraph are
-included on all such copies and derivative works.  However, this
-document itself may not be modified in any way, such as by removing
-the copyright notice or references to the Internet Society or other
-Internet organizations, except as needed for the purpose of
-developing Internet standards in which case the procedures for
-copyrights defined in the Internet Standards process must be
+included on all such copies and derivative works.
+However, this document itself may not be modified in any way, such as
+by removing the copyright notice or references to the Internet
+Society or other Internet organizations, except as needed for the
```

Comment:
If you know reviewer time is extemely limited, and you know that there are meaningful content patches to the manual pending review, is it appropriate to edit docs with scripts and not even bother testing or thinking critically about the result?